### ISSUE TITLE
ipv6 all host [::] inline mode with error

### ISSUE THREAD

**Issue by NewFuture** [webpack/webpack-dev-server#672]
**I'm submitting a bug report**

**webpack and webpack-dev-server version:**
webpack: 1.13.3
webpack-dev-server: 1.16.2

**Please tell us about your environment:**
Linux 
Running server via CLI  

```
webpack-dev-server --host "::" --hot --inline
```

**Current behavior:**

```
Uncaught SyntaxError: The URL 'http:/::' is invalid(…)
```

**Expected/desired behavior:**
work as `--host 0.0.0.0`

this bug coursed by  parsing the ipv6 IP address . 
the ipv6 url is parsed as `:::8080` ,
which should be
```
{ip,port} => [ip]:port
{ ::,8080} => [::]:8080
```

**Comment by SpaceK33z** [comment]
Thanks for your clear bug report. This was fixed in webpack-dev-server v2, which is currently in beta. See PR #644.

**Comment by NewFuture** [comment]
sorry, it does not work .😂 

see here:  https://github.com/webpack/webpack-dev-server/blob/master/bin/webpack-dev-server.js#L276

**Comment by SpaceK33z** [comment]
@NewFuture, oh haha okay. Fair enough. Would you be willing to send a PR to fix this? I don't have much time to fix this at the moment.

**Comment by NewFuture** [comment]
OK,  I have fixed it. 😸 
I found another bug in ipv6, and fixed both.

see this #673 

But , i think some  test examples for ipv6 should be added.

🐱

**Comment by SpaceK33z** [comment]
Released in `2.1.0-beta.10`!

**Comment by NewFuture** [comment]
😀 nice!
